lib/src/metta/runner/modules/mod.rs

```diff
@@ -209,18 +212,21 @@ impl MettaMod {
     }
 
     fn export_all_tokens_into(&self, target_mod: &MettaMod, metta: &Metta) -> Result<(), String> {
+
+        // println!("export_all_tokens_into from {:?}", self.name());
+        // println!("export_all_tokens_into to {:?}", target_mod.name());
+        // print!("self tokens count:");
+        //target_mod.tokenizer().borrow().print_tokens_count();
         if self.name() == "corelib" {
-            register_runner_tokens(&mut *target_mod.tokenizer().borrow_mut(), target_mod.tokenizer().clone(), &DynSpace::with_rc(target_mod.space.clone()), metta);
-            register_common_tokens(&mut *target_mod.tokenizer().borrow_mut(), target_mod.tokenizer().clone(), &DynSpace::with_rc(target_mod.space.clone()), metta);
-            register_rust_stdlib_tokens(&mut *target_mod.tokenizer().borrow_mut());
+            register_all_corelib_tokens(&mut *target_mod.tokenizer().borrow_mut(), target_mod.tokenizer().clone(), &DynSpace::with_rc(target_mod.space.clone()), metta);
+          
         } else {
-            // Get the tokenizer associated with the dependent module
-            let dep_tokenizer = self.tokenizer().clone();
-
+            let dep_tokenizer = self.own_tokenizer().clone();
```

Comment:
Clone is not needed here, taking into account we will clone it in the next line.
```suggestion
            let dep_tokenizer = self.own_tokenizer();
```